### PR TITLE
moves constant to config & fixes tip image preview

### PR DESCRIPTION
**Pull request by kenodressel**

### CODE REVIEW

**Comment by kenodressel** [comment]
Some previews are still broken but this is a backend issue and not related to this PR

File: src/utils/backend.js [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
 
 export default class Backend {
 
   BACKEND_URL;
   MAINNET_URL;
 
   constructor() {
-    this.BACKEND_URL = typeof Cypress !== 'undefined' ? 'http://localhost' : "https://raendom-backend.z52da5wt.xyz";
-    this.MAINNET_URL = 'https://mainnet.aeternal.io/'
+    this.BACKEND_URL = BACKEND_URL;
```
Comment (thepiwo) [inline src/utils/backend.js]: no need for this

File: src/components/tipRecords/TipRecord.vue [hunk 2]
```diff
@@ -70,6 +71,9 @@ export default {
       if (!this.isPreviewToBeVisualized(this.tip)) return '';
       return this.tip.preview.description ? this.tip.preview.description : this.tip.preview.title;
     },
+    tipPreviewImage() {
+      return this.isPreviewToBeVisualized(this.tip) ? `${BACKEND_URL}${this.tip.preview.image}` : '';
```
Comment (thepiwo) [inline src/components/tipRecords/TipRecord.vue]: maybe build url in backendinstance, to have all urls summarized there

Comment (kenodressel) [reply src/components/tipRecords/TipRecord.vue]: I guess we could do @mradkov whats your opinion?

Comment (mradkov) [reply src/components/tipRecords/TipRecord.vue]: Yes, I agree with @thepiwo, if we are keeping the url there having all urls summarized there is reasonable.

**Comment by thepiwo** [comment]
@kenodressel can you resolve conflicts and run linter?

**Comment by kenodressel** [comment]
rebased & fixed lint errors, ready to merge :)

**Pull request merged**